lib/sms/sms_submit.c

```diff
@@ -130,6 +273,8 @@ static void sms_submit_print_error(int err)
  * @param[in] encoded_data_size_septets Encoded User-Data size in septets.
  * @param[in] message_ref TP-Message-Reference field in SMS-SUBMIT message.
  * @param[in] udh_str User Data Header. Can be set to NULL if not desired.
+ * @param[in,out] sca_field SMS Service Center address field of the SMS pdu within CMGS AT command.
```

Comment:
Isn't this parameter `in` only in this function?

Reply:
Yep, it's `in` here.